package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gsf-js-client-sdk",
-  "version": "2.0.5",
+  "version": "2.0.4",
```

Comment:
Are we going back a version?

Reply:
We can just leave this at 2.0.5 - I'll take care of bumping and publishing.

Reply:
Whoops! 2.0.5 is correct right?